src/test/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println(getName() + " started her work.");
```

Comment:
The message should indicate what type of `Machine` is working. Instead of using 'her', specify that a `Bulldozer` is working to make the log message more informative.